static/css/app.v1.css

```diff
@@ -755,27 +765,22 @@ a.back-btn {
   }
 }
 
-.js-active .js-click thead,
-.js-active .hide-columns tbody {
+.hide-columns tbody {
   cursor: pointer;
 }
 
-.js-click .rowid {
-  display: none;
-}
-
-.js-active .hide-columns .Link,
-.js-active .hide-columns .rowid,
-.js-active .hide-columns .suffix,
-.js-active .hide-columns .system,
-.js-active .hide-columns .agency,
-.js-active .hide-columns .class_code,
-.js-active .hide-columns .pay_rate,
-.js-active .hide-columns .annual_salary,
-.js-active .hide-columns .regular_earnings,
-.js-active .hide-columns .overtime_earnings,
-.js-active .hide-columns .other_earnings,
-.js-active .hide-columns .term_date {
+.hide-columns .Link,
```

Comment:
.js-active was there so users without JS would still see the columns, since they can't toggle things.

Reply:
Oh of course! I'll add those back in.